wiki/osu!supporter/en.md

```diff
@@ -45,23 +41,34 @@ Laconic: Donation is used to sustain osu! overall
 | 12     | 26 (-46% max)                        |
 | 24     | 52 (-46% max)                        |
 
--   Costs are by individual account. There are no promotions for group gifting.
+Note:-
+
+- Costs are by per individual account. There are no promotions for group gifting.
 
 ### Decide
```

Comment:
maybe change this to `### Gifting`
`### Decide` doesn't seem to fit this too well :/